### PR TITLE
three minor fixes: document /dev/console/SAK handling + another cgroups wording fix + document User= default

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: man/systemd.exec.xml [hunk 1]
```diff
@@ -141,8 +141,10 @@
         <term><varname>Group=</varname></term>
 
         <listitem><para>Set the UNIX user or group that the processes are executed as, respectively. Takes a single
-        user or group name, or numeric ID as argument. If no group is set, the default group of the user is used. This
-        setting does not affect commands whose command line is prefixed with <literal>+</literal>.</para></listitem>
+        user or group name, or numeric ID as argument. If not user is explicitly set the user the invoking service
```
Comment (keszybz) [inline man/systemd.exec.xml]: "If the user is not explicitly set"? Also "inherited" is jargon-y. We should also say that this only works for the system instace. What about:

> <strike>For the system systemd instance, the default is "root". For the user instance, switching users is not supported, so the only setting that is valid is to specify the same user that the manager instance is running under.</strike>
> 
> For the system systemd instance and for root's user instance, the default is "root", but User= can be used to specify a different user. For other instances, switching users is not supported, so the only setting that is valid is the same user that the manager instance is running under.

Comment (evverx) [reply man/systemd.exec.xml]: > For the user instance, switching users is not supported

What about `user@0.service`:

```
root       250  0.2  1.3  50584  6512 ?        Ss   04:10   0:00 /lib/systemd/systemd --user
```

?

**Comment by evverx** [comment]
```
TEST RUN: Job-related tests [FAILED]
Makefile:8: recipe for target 'run' failed
make: Leaving directory '/tmp/autopkgtest.9Ry6QI/build.paG/systemd-debian/test/TEST-03-JOBS'
make: *** [run] Error 1
```

Unrelated to this PR, of course. But seems like we break something.
https://github.com/systemd/systemd/pull/4331#issuecomment-252830790

File: man/systemd.exec.xml (hunk 1 above)
Comment (keszybz) [reply man/systemd.exec.xml]: Wasn't systemd changed to basically ignore User= in the user instance?

Comment (keszybz) [reply man/systemd.exec.xml]: Hm, I was wrong. Testing indeed shows that User= works under user@0.

So some different wording should be used.

**Comment by poettering** [comment]
I force pushed a new version now. Rebased and incorporating the suggested changes. That said, I also reworded the paragraph in other ways.

I tried to avoid the term "systemd instance", and try to say "system service manager" and "user service manager" instead, simply as I think "systemd" is a bit too unspecific, as it could refer to just PID 1 (which is what is mean here) or the whole project/package (which isn't).

I also used the wording "not permitted" rather than "not supported" regarding user switching in the --user instance, since it's really about permissions here, not about support...

**Pull request merged**